docs/rules/use-string-join.md

```diff
@@ -75,4 +75,39 @@ String result = String.join(",", values);
 ```
 
 
+### Bytecode JDK 1.8 
+
+__Pre__
+```java
+public void original(List<String> values) {
+    String result = values.stream().collect(Collectors.joining(","));
```

Comment:
This is pretty much the output from the 'Replace For-Loop with Stream::collect(Collectors.joining())' rule. It would be nice if we had a part that described which rules further improve on the output of another rule.